docs/containerd-env-setup.md

```diff
@@ -56,28 +56,37 @@ $ sudo tee /etc/nydusd-config.json > /dev/null << EOF
 EOF
 ```
 
-Note:
+Please refer to the nydusd [doc](./nydusd.md) to learn more options.
+
+⚠️ Note:
+
+- The `device.backend.config.scheme` is registry url scheme, leave it empty to automatically detect, otherwise specify to `https` or `http` according to your registry server configuration.
+- The `device.backend.config.auth` is base64 encoded `username:password`, it is required by `nydusd` to lazily pull image data from registry which is authentication enabled, the nydus snapshotter will automatically read docker login auth from the configuration `$HOME/.docker/config.json`, otherwise please copy it to replace `YOUR_LOGIN_AUTH=`.
+- The `device.backend.config.skip_verify` allows to skip insecure registry https certs, only set to `true` if necessary.
```

Comment:
- The device.backend.config.skip_verify allows you to skip insecure HTTPS certificates for the registry. Only set this to true if necessary. Note that this can compromise the security of your connection to the registry, so you should only use it if you are sure that it is safe to do so.

Reply:
How about:

This  `device.backend.config.skip_verify` allows you to skip the insecure https certificate checks for the registry, only set it to `true` when necessary. Note that enabling this option is a security risk for the connection to registry, so you should only use this when you are sure it is safe.